.gitignore

```
@@ -57,7 +57,7 @@ ptop*.json
 /coverage-report
 
 # pyenv
-/.python-version
+**/.python-version
```

Removing the `/` prefix would also work, and I believe is the more typical way to ignore a file anywhere in the repo.

```suggestion
.python-version
```